docs/npwd/start/Install.md

```diff
@@ -0,0 +1,102 @@
+---
+id: installation
+title: Install and Setup
+sidebar_label: Installation
+---
+
+### Third-Party Resource Dependencies
+If you wish for NPWD to have full functionality. The following resources are required to be
+installed and started before NPWD.
+
+1. [screenshot-basic](https://github.com/project-error/screenshot-basic) - **Make sure to use the exact version, as linked.**
+2. [pma-voice](https://github.com/AvarianKnight/pma-voice) - Support for other VOIP resources is planned (`toko-voip`) etc
```

Comment:
Iirc, support for other voices is no longer planned as they’re all inferior to pma.